src/containers/SignupMain.js

```diff
@@ -6,7 +6,6 @@ import * as actions from "actions/SignupMain";
 
 const mapStateToProps = (state, props) => {
   return {
-    is_fetching: state.config.is_fetching,
```

Comment:
I just want to double check that it's ok to go beyond commenting this out here and fully remove it

Reply:
The signup app does not need to know whether the app is fetching any more; just the buttons need to know